esrally/config.py

```diff
@@ -867,6 +844,38 @@ def migrate(config_file, current_version, target_version, out=print, i=input):
         current_version = 14
         config["meta"]["config.version"] = str(current_version)
 
+    if current_version == 14 and target_version > current_version:
+        # Be agnostic about build tools. Let use specify build commands for plugins and elasticsearch
+        # but also use gradlew by default for Elasticsearch and Core plugin builds, if nothing else has been specified.
+
+        def warn_if_plugin_build_task_is_in_use(config):
+            if "source" not in config:
+                return
+            for k, v in config["source"].items():
+                plugin_match = re.match('^plugin\.([^.]+)\.build\.task$',k)
+                if plugin_match != None and len(plugin_match.groups()) > 0 :
+                    plugin_name = plugin_match.group(1)
+                    new_key = "plugin.{}.build.command".format(plugin_name)
+                    out(
+                        """
+                        WARNING:
+                          The build.task property for plugins has been obsoleted in favor of the full build.command.
+                          You will need to edit the plugin [{}] section in your rally ini file and change from:
```

Comment:
Maybe change the wording to: " ... section in ~/.rally/rally.ini"? You should be able to get the path via `config_file.location`.